Samotny Pr bez przyjaciół z wyjątkiem stwórcy i Amelki

Selerek opened the pull request.

Selerek committed:
created project

Selerek committed:
header works

Selerek committed:
header works 100%

chivasioooo

Selerek committed:
gradient works

NoWy BatMan

Selerek committed:
footer works!

mordo bardziej blueface

Selerek committed:
footer działa skibidi

nay sos ciuchy i Amelia

amystas commented:
zmień nazwy klas "prawy" i "lewy" na angielskie "right" i "left"

Selerek commented:
> zmień nazwy klas "prawy" i "lewy" na angielskie "right" i "left"

dzieki mordo jak bede na chacie to zobacze,
zaraz jak na chate mi wyslal????

Selerek committed:
Yuji Itadori

left right goodnight

Selerek committed:
mapa works(ledwo)

dziala ale jak trytytka po 10 latach na słońcu

Selerek committed:
mapa created mocnooo

nadchodzi pieeklo no bo kazda dziewczynka chcialaby zostac influencerką

Selerek committed:
mapka sciaga JSONa z linka

wszystkoo czegoo tylko dotkne to staje sie złootem to staje sie złootem MIDAS MIDAS MIDAAAAS

Selerek committed:
Cała mapa sie super generuje póki link działą

I BELIEVE IN JOE HENDRY

amystas commented on HaveASeat/src/app/mapa/mapa.component.ts:
```diff
@@ -1,19 +1,61 @@
+
+  getBorder(roomId: number, positionY: number, positionX: number): string {
+    const room = this.rooms.find(r => r.id === roomId);
+    const cell = room?.cells.find(c => c.positionX === positionX && c.positionY === positionY);
```
W przyszłości zmień na wybieranie komórek po ich id (masz je w jsonie)

Selerek committed:
dziala z bledem

Selerek committed:
miejsca sie generują wszystkie

to NoWy BaTmAn

Selerek committed:
prawie dziala obracanie

Selerek committed:
krzesełka sie obracają

ciekawe czy odczytasz, przez ciebie nie spie dzieki tobie zasypiam, mamy wspolne zdjecia zapisane na kliszach, mysle o niej czesciej ona o narkotyckach. MIJAMY SIE NA MIESCIE, POWIERZE ROBI SIE CIEZKIE, chcialbym jej bardziej nie cierpiec, nie podejde bo czekam aż mi przejdzie. Ona jest bez serca, ma je na butach,

Selerek committed:
commit kontrolny 123

tak tak tak spada jedna z gwiazd cicho suwa sie z wysoka, może dzis wreszcie poweim ci powiem jak cie bardzo kocham CALY SWIAT NIECH WIE ZE JA KOCHAM CIE  ze bez ciebie zyc nie umiem juz, popatrz w serca me bo dzis na ich dnie znajdziesz tą ukryą orawde, GWIAZDA W GORZE LSNI pierwszym mym zyczeniem jestes ty spadajaca gwiuaza spelnia sny ktorych glowna role grasz ty

amystas commented on HaveASeat/src/app/mapa/mapa.component.css:
```diff
@@ -0,0 +1,59 @@
+  background-position: center top;
+  content: "";
+  position: absolute;
+  z-index: 10;
```
czemu 10? nie ma innych elementów o atrybucie z-index

Selerek replied:
nie ma żadnego z indexu tutaj

amystas commented on HaveASeat/src/app/mapa/mapa.component.ts:
```diff
@@ -0,0 +1,101 @@
+    );
+  }
+
+  check(desk :Desk, cell :Cell) :boolean {
```
nazwa funkcji niejednoznaczna

Selerek replied:
funkcja testowa 123

amystas commented on HaveASeat/src/assets/images/logo.png:
projekt w figmie przewiduje inne logo

Selerek replied:
to sie zrobi w innej fazie

amystas commented on HaveASeat/src/app/mapa/mapa.component.ts:
```diff
@@ -0,0 +1,101 @@
+    
+    switch (chairPosition) {
+      case 1:
+        console.log("prawo")
```
console.log() możesz zamienić na komentarz w kodzie

Selerek replied:
spox

amystas commented on HaveASeat/src/app/mapa/mapa.component.ts:
```diff
@@ -0,0 +1,101 @@
+    forkJoin({
+      rooms: this.mapaService.getRooms(),
+      //desks: this.mapaService.getDesks()
+    }).subscribe(
```
przestarzałe użycie funkcji, wykorzystaj complete, next i error
dalsze info: https://v17.angular.io/guide/observables#defining-observers

Selerek replied:
nie ma bata że to zmieniam

Selerek committed:
usprawnienia i stylizacja

wychoduje sobie drzewko na betonie podlewanej łezką wybacz jego owoce kiedys zwiedną i mam problem żeby je dosiegnać

Selerek committed:
ebebe

123

amystas committed:
nie ma za co

Selerek committed:
wyswietlaja sie rezerwacje

ale w sumie przepraszam Batory był tylko symbolem a bluze musialem zalozyc i w sumie kazdy normalny to zrozumial, oprocz krystyny pawłowicz kretynie po co to robisz piszą mi w dm sfrustrowane matki o demoralizacje ich dzieci i rodzin ja tylko mowie jak zyją to wy mówiłyście jak mają to robisz

Selerek committed:
krzeslo jak klikniesz to sie na niebieskie zmienia, zdjecia do poprawy

co by bylo gdybym go uderzyl troche bardziej i to chyba bylby juz moj ostatni albul gdyby te depresje mialby leczyc Robert Pasut. NADCHODZI PIEEEEKLO NO BO KAZDA DZIEWCZYNKA CHCIALABY ZOSTAC INFLUENCERKAAAA a myslalem ze zyjemy juz w ciezkich czasach ONE NADEJDAAAA NO BO KAZDA DZIEWCZYNKA CHCIALABY ZOSTAC INFLUENCERKĄĄĄĄĄĄĄĄĄĄĄĄĄĄĄĄĄĄĄĄ

Selerek committed:
krzeselka pozmieniane

d jak dycha wiele imion ma

amystas committed:
one clicked at a time

amystas committed:
user reservation is blue

Selerek committed:
user can reserve only one desk, database updates

Selerek committed:
minor fix

nikt nie chce wiedziec co tam było miedzy nami, ale ja nie chce zapomniec lata, smialismy sie razem tak pijani, a pytałaś mnie o ktorej musze wracać. Mysle o niej tutaj cały czas, zapomnialem jak to było ale pamietam ten smak. Morza szum to na jej ustach biały pasek w oczach, opalona skóra muszę lato uratować, nie chce żeby wróciła do niego prosze zostań. Mówiła mi they dont really care about us uuuuuuuuuuuuu they dont really care about us uuuuuuuuuuuu they dont really care about us

Selerek committed:
nowy wyglad i mozna cofac rezerwacje

aiaiai im your little butterfly

Selerek committed:
wszystko ładnie wystylowane

uouo autos moda y rock and roll uouo presta cibi binacjo

Selerek committed:
krzesełka równe skibidi

jksbvyghnwrehtupcyerhgiudrhwoyjgiudfytfh

Selerek committed:
add cursor changes

Co-Authored-By: AmyStas <95294347+amystas@users.noreply.github.com>

Selerek committed:
minor fix

Co-Authored-By: AmyStas <95294347+amystas@users.noreply.github.com>

Selerek committed:
switching users

Co-Authored-By: AmyStas <95294347+amystas@users.noreply.github.com>
